src/index.css

```diff
@@ -11,3 +11,159 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*Page styling - background and albums*/
+body {
+  background-color: rgb(0, 0, 0);
+  box-sizing: border-box;
+  text-align: center;
+}
+
+/*Have trouble with the grid being to large - have to scroll horizontly - would fix if I had more time*/
+.albumWrapper {
+  margin-top: 50px;
+  display: grid;
+  grid-gap: 40px;
+  grid-template-columns: repeat(1, 1fr);
+  margin-bottom: 50px;
+}
+
+header {
+  color: whitesmoke;
+  margin-top: 28px;
+  font-size: 25px;
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em 0em;
+  font-weight: 700;
+}
+
+/*ALBUM NAME AND ARTIST NAME and HOVER*/
+
+.albumName {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color: #ffffff;
+  font-weight: bold;
+}
+
+.albumName:hover {
+  text-decoration: underline;
+}
+
+.artistName {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color: #a0a0a0;
+  font-weight: bold;
+}
+
+.artistName:hover {
+  text-decoration: underline;
+}
+
+a {
+  text-decoration: none;
+}
+
+
+/*ALBUM AND ICON BUTTONS*/
+
+.coverImage {
+  position: relative;
+  display: inline-block;
+  transition: 0.3s ease;
+}
+
+.ikonButtons {
+  /*placement*/
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
```

Comment:
I wonder if using both 'position: absolute' and 'display: flex' is necessary? Don't both of them just position the content? And seeing you have 0 in both the values regarding the position, I'm not sure what it does.